src/extensions/replay/config.ts

```diff
@@ -68,7 +68,7 @@ const removeAuthorizationHeader = (data: CapturedNetworkRequest): CapturedNetwor
     return data
 }
 
-const POSTHOG_PATHS_TO_IGNORE = ['/s/', '/e/', '/i/vo/e/']
+const POSTHOG_PATHS_TO_IGNORE = ['/s/', '/e/', '/i/v0/e/', '/i/vo/e/', '/decide/', '/static/recorder-v2.js']
```

Comment:
the other option is to ignore the whole domain rather than just the paths.

Reply:
i was wondering about that... once the dust has settled we should have ingestion domain and non-ingestion domains and then we can ignore the ingestion domain